lib/croppable/model.rb

```diff
@@ -42,7 +42,8 @@ def #{ name }=(croppable_param)
                   self.#{ name }_croppable_data = Croppable::Datum.new(croppable_param.data.merge(name: "#{ name }"))
                 end
 
-                to_crop_croppable[:#{ name }] = self.#{ name }_croppable_data.updated_at_previously_changed? || self.#{ name }_croppable_data.new_record?
+                self.#{ name }_croppable_data.updated_at_previously_changed? || self.#{ name }_croppable_data.new_record?
+                  to_crop_croppable[:#{ name }] = { host: croppable_param.host }
```

Comment:
What do you think about getting the host in here form the request without using the croppable param?

Reply:
Sure, I didn't see an easy alternative since it's the only thing passed through to the model. What's an alternative?